### DISCUSSION TITLE
[Master HamsterKombat Bot][ERROR] Unable to get daily keys mini game.

### DISCUSSION THREAD

**Discussion by devCarlosHenSil** [masterking32/MasterHamsterKombatBot#254] [General]
Hello, a good day everyone, I would like to know if you are having problems solving the Puzzle game since they added Hexa Puzzle, mine simply no longer collects the Puzzle game key, I would like to know how to solve this.
thank you very much!
![Captura de tela 2024-08-28 114129](https://github.com/user-attachments/assets/c9f7638c-a556-4109-a97c-7f5f67c491bc)

**Comment by masterking32** [comment]
https://github.com/masterking32/MasterHamsterKombatBot/issues/236
